### PR TITLE
Update CONTRIBUTING.md

### LINKED ISSUE #5992: XLF files don't need to be touched except for releases
**Do you want to request a *feature* or report a *bug*?**
feature
**What is the current behavior?**

Contributors tend to alter xlf files when submitting pull requests.

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem.**

**What is the expected behavior?**

Help guide contributors into not touching xlf files because it is done by TranslationApp and release generation.

**Environment you encounter the issue:**

- Git Extensions 3.00.00.4433
- Build fca7cf228b481ee8c1b779cf7b882ccdfbdcd1bc
- Git 2.20.1.windows.1
- Microsoft Windows NT 6.3.9600.0
- .NET Framework 4.7.3260.0
- DPI 96dpi (no scaling)

**Did this work in previous version of GitExtensions (which)?**

### PR DESCRIPTION
**Pull request by vbjay**
Point out XLF files do not need to be touched

Fixes #5992

## Proposed changes
- Update CONTRIBUTING.md to point out xlf files do not need to be modified.

## Test methodology 

NA

## Test environment(s) 
NA

### CODE REVIEW

**Comment by RussKie** [comment]
Ideally we solve this via automations, #5189

**Comment by RussKie** [comment]
Thank you

**Comment by vbjay** [comment]
I saw that.  I figured until that was done, I figured we could guide them to not making those changes.

**Comment by gerhardol** [comment]
With the current workflow, I would have preferred if translation changes always included running the translation app.
 * @RussKie had issues running the app. 
 * #5976 made output garbage, seen first after updating the translation app.
 * #5729 about adding garbage translatable strings (in addition to the issue later seen in #5976)
  
The two last issues could have been detected by the maintainer that updated the translation, but RussKie was quite pressed at that time. It is a very serious problem if he fail to see it.

Earlier updates could have detected the issues in reviews or by usage (as the problems would be seen).

But the update is not contradicting earlier xlf refreshes.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -56,6 +56,8 @@ Specifically:
 When filing a pull request, you should be prepared to answer questions about your changes
 and to perform additional work on the changes in response to review feedback.
 
+.xlf files do not need to be modified.  The English xlf is generated by the TranslationApp and then uploaded to Transifex when a release is generated.  The other languages are generated from Transifex site.
```
Comment (gerhardol) [inline CONTRIBUTING.md]: Sidenote: This is using "American" double space after periods where most using single space.
On the other hand, I would propose to use US English consistently, not to mix.
(The documentation is mostly in British English now.)
Maybe should be added to a style guide.

**Comment by RussKie** [comment]
> With the current workflow, I would have preferred if translation changes always included running the translation app.

I fully agree, however only if we can make it transparent to developers. 
If not, it would be a burden for everyone...

**Comment by gerhardol** [comment]
I would prefer to have running TranslationApp as a recommendation, not to discourage it.

**Comment by RussKie** [comment]
:+1:

**Pull request merged**